### DISCUSSION TITLE
i facing this bug

### DISCUSSION THREAD

**Discussion by Danishshaikh9702** [authelia/authelia#5752] [Q&A] [answered]
2023/08/02 10:12:24 error when serving connection "127.0.0.1:3700"<->"127.0.0.1:54646": error when reading request headers: EOF. Buffer size=84, contents: ",\"userId\":\"123\",\"messageId\":\"VeKx6_V1Tf6s3AZnS2F6Aw-0000000100\",\"processType\":\"web\"}"

**Comment by james-d-elliott** [comment, marked as answer]
Looks like some client is making a request and abruptly closing the connection before sending request headers or half way through sending them.

**Reply by Danishshaikh9702** [reply]
then  what i have do to solve this issue

**Reply by james-d-elliott** [reply]
You can ignore it (as it's just a log message and you've not described any negative effect) or track down the client which is performing the bad request.
